### PR TITLE
Footer added

### PR DESCRIPTION
**Pull request by kamalchakma1**
Hi, I have added footer section. Please check

### CODE REVIEW

File: src/Pages/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            {/* === Footer Left Section === */}
+            <Grid item lg={4} xs={12}>
+                <div className="grid_container__box1"> 
+                    {/*=== Logo Cotainer === */}
+                   <div className="grid_container__box1__logo_container">
+                    <div><img src={"images/GDSC LOGO FOOTER.png"} height={"45px"} alt="gdsc"/></div>
+                    <div><LocationOnIcon/></div>
+                    <div><EmailIcon/></div>
+                    </div> 
+                    {/*=== Text Cotainer === */}
```
Comment (Yashsharma1911) [inline src/Pages/Footer/Footer.jsx:45]: nice work with comments 👏

**Review by Yashsharma1911 — COMMENTED** [review]
It will be helpful if you can provide a screenshot of the changes

File: src/Pages/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                        <ThemeProvider theme={theme}>
+                        <Typography variant="h5">Google Developer Student Club</Typography>
+                        </ThemeProvider>
+                        <ThemeProvider theme={theme}>
+                          <Typography variant="subtitle2">ITM Guwalior</Typography>
+                        </ThemeProvider>                       
+                        
+                    </div>
+                    <div class="grid_container__box1__text_container__address">
+                        <ThemeProvider theme={theme}>
```
[the remark this answers is not in the document]
Comment (kamalchakma1) [reply src/Pages/Footer/Footer.jsx:58]: yes we can use it at top level, It was already there in top level so I didn't touch that one. Feel free to remove it.

**Comment by kamalchakma1** [comment]
Here the screenshot - 
Footer desktop

File: src/Pages/Footer/Footer.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Yashsharma1911) [reply src/Pages/Footer/Footer.jsx:58]: thanks for confirming @kamalchakma1 can you do that it would be helpful

**Comment by Yashsharma1911** [comment]
> Here the screenshot - Footer desktop

this looks pretty 😎

File: src/Pages/Footer/Footer.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kamalchakma1) [reply src/Pages/Footer/Footer.jsx:58]: ok

**Comment by Yashsharma1911** [comment]
as we are in a sprint to make and build the website as soon as possible I am merging this PR, for other updates you can raise a new PR for this

**Review by Yashsharma1911 — APPROVED** [review]

**Comment by kamalchakma1** [comment]
Hi @Yashsharma1911 Sorry I  am little busy this week so I'm not able to do that changes. And Thanks for merging.

**Comment by Yashsharma1911** [comment]
also, let's get you ramped up @kamalchakma1 we are in need of some new issues for new contributors who may expect to come after the 31 of march because of an event so it will be helpful if you can take this milestone 

you just need to look into the codebase and website if you find something needs to be fixed create an issue for that

**Comment by kamalchakma1** [comment]
Sure I love to take this milestone.

**Comment by Yashsharma1911** [comment]
That's great @kamalchakma1

**Pull request merged**